bin/commands/new.sh

```diff
@@ -24,39 +22,37 @@ function new_cmd {
     echo "$image_dir"
   }
 
-  function create_image_metadata {
+  function create_image_version_dir {
     local dir=$1
+    local version=$2
 
-    local metadata_path="$dir/meta.sh"
-    echo "#!/bin/bash" > "$metadata_path"
-    chmod +x "$metadata_path"
+    if [[ -z "$version" ]]; then
+      version="latest"
+    fi
```

Comment:
`latest` should be always set through a symbolic link, so use him as a default here is wrong